Bartosz/index.js

```diff
@@ -0,0 +1,332 @@
+// Programming Basics: For Loops
+
+// These exercises are designed for practising "for" loops. Print all your results to the console.
+
+// 1. **Addition.** 
+// Write a program to add up the numbers 1 to 20.
+
+let sum = 0;
+for (let i = 1; i <= 20 ; i++) {
+    sum = sum + i;
+}
+
+console.log(`sum = ${sum}`); // sum = 210
+
+// 2. **There are i bottles of beer on the wall.** 
+// Write a program that will output, "There is one bottle of beer on the wall." "There are two bottles of beer on the wall" up until there are five bottles. 
+
+for (let bottles = 0; bottles <= 5; bottles++){
```

Comment:
we need if statement to manage
if one "There is one bottle of beer on the wall."
more than one  "There are two bottles of beer on the wall"